test/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -168,6 +169,9 @@ public void SetupPredictBenchmarks()
 
         [Benchmark]
         public void PredictIrisBatchOf5() => _trainedModel.Transform(_mlContext.Data.LoadFromEnumerable(_batches[2]));
+
+        [Benchmark]
```

Comment:
I'm a little bit confused about this benchmark. If it's meant to actually create a benchmark of how fast it does evaluation, then you'd actually need to call `_evaluator.Evaluate()`, the way it is now, it simply benchmarks how quickly an evaluator is instantiated, so the name of the benchmark is a little bit misleading.

Reply:
facepalm